client/extensions/woocommerce/app/stats/stats-navigation/index.js

```diff
@@ -27,16 +26,16 @@ const StatsNavigation = props => {
 		subscriptions: translate( 'Subscriptions' ),
 	};
 	return (
-		<SectionNav selectedText={ sectionTitles[ section ] }>
+		<SectionNav selectedText={ types[ type ] }>
 			<NavTabs label={ translate( 'Stats' ) }>
-				<NavItem path={ `/store/stats/${ slug }` } selected={ section === 'orders' }>
-					{ sectionTitles.orders }
+				<NavItem path={ `/store/stats/${ slug }` } selected={ type === 'orders' }>
```

Comment:
Maybe I'm missing something, but shouldn't this `path` be `/store/stats/orders/${ slug }`? Same for the other `<NavItem>` elements.